pkg/config_test.go

```diff
@@ -0,0 +1,64 @@
+package pkg
+
+import (
+	"io/ioutil"
+	"os"
+	"os/user"
+	"testing"
+)
+
+func TestGetregistryParams(t *testing.T) {
+	usr, err := user.Current()
+	if err != nil {
+		t.Logf("could not retrieve current user: %v", err)
+		t.SkipNow()
+	}
+	if usr.Uid != "0" {
+		t.Logf("current user does not have UID of zero (got %s) "+
+			"so cannot create registry cred location, skipping", usr.Uid)
+		t.SkipNow()
+	}
```

Comment:
this is just checking if we can create files for testing.As we have to create the files on specified for testing i don't think this should fail.
this is the exact copy of the things we are doing in the builder.